day_6/exercises/burrito.rb

```diff
@@ -12,20 +12,27 @@ def initialize(protein, base, toppings)
     @toppings = toppings
   end
 
-  def add_topping
-  end
+def add_topping(corn)
+  self.toppings.push(corn)
+end
 
-  def remove_topping
-  end
+def remove_topping(guacamole)
+  self.toppings.delete(guacamole)
+end
 
-  def change_protein
-  end
+def change_protein(protein)
+  @protein = protein
+ end
```

Comment:
Good work on the `change_protein` method that directly updates an `@instance_variable` using an argument.

Question regarding your toppings: Will the topping updated always be `corn`? Or always delete `guacamole`? What might be a better name for your argument for these methods?

Note on `self`: Rarely will you ever need to use `self` - especially in mod 1. What might be a different way to access the toppings attribute directly?